backends/advanced/src/advanced_omi_backend/services/obsidian_service.py

```
@@ -0,0 +1,447 @@
+        # backends/advanced/src/advanced_omi_backend/services/obsidian_service.py
+        self.CURRENT_DIR = Path(__file__).parent.resolve()
+
+        # Load configuration strictly from standard locations
+        # Prefer /app/config.yml inside containers (mounted by docker-compose)
+        # Fallbacks handled by shared utility
+        config_data = load_root_config()
+
+        # Get model configurations using shared utility
+        llm_config = get_model_config(config_data, 'llm')
+        if not llm_config:
+            raise ValueError("Configuration for 'defaults.llm' not found in config.yml")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)